pkg/ddevapp/mutagen_test.go

```diff
@@ -63,6 +63,10 @@ func TestMutagenSimple(t *testing.T) {
 
 	assert.True(dockerutil.VolumeExists(ddevapp.GetMutagenVolumeName(app)))
 
+	mutagenConfig, err := os.ReadFile(ddevapp.GetMutagenConfigFilePath(app))
+	require.NoError(t, err)
+	require.Contains(t, mutagenConfig, app.GetUploadDir())
```

Comment:
Shouldn't it check for each item in app.GetUploadDirs()?

Reply:
Also thought about that but as there is only one item anyway I decided to go this simpler path.